readme.md

```diff
@@ -21,8 +21,8 @@ Follow
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_stars/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_stars/report/html_report/)
+- [DEMO LINK](https://rwlp.github.io/layout_stars/)
+- [TEST REPORT LINK](https://rwlp.github.io/layout_stars/report/html_report/)
 
 ❗️ Copy this `Checklist` to the `Pull Request` description after links, and put `- [x]` before each point after you checked it.
```

Comment:
You should mark the checklist and copy it entirely on your pull request comment, like this solution here: https://github.com/mate-academy/layout_stars/pull/3962#issue-2061470202

It helps us mentors to evaluate your solution more easily